apps/sla/sla.yml

```diff
@@ -44,11 +44,12 @@ sla:
 name: Critical Compliance Priority
 description: >
   SLA policy applied to critical impact on
-  compliance priority module and component
+  compliance priority module, component and stream
 conditions:
+  tracker:
+    - is compliance priority
```

Comment:
I would be for alphabetical ordering of the dictionary attributes from top to bottom if there is no reason not to do it that way - eg. that `tracker` would be some kind of title etc.

Reply:
Fixed (put `tracker:` below `flaw:`)